archivebox/core/models.py

```diff
@@ -72,7 +72,7 @@ def link_dir(self):
 class SnapshotResult(models.Model):
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
 
-    snapshot = models.ForeignKey(Snapshot)
+    snapshot = models.ForeignKey(Snapshot,on_delete=models.CASCADE)
```

Comment:
Without getting into what the code change does:
```suggestion
    snapshot = models.ForeignKey(Snapshot, on_delete=models.CASCADE)
```